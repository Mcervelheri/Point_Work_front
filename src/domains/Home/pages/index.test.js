import React from 'react';

import { createMemoryHistory } from 'history';
import { Provider } from 'react-redux';
import { Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import configureMockStore from 'redux-mock-store';

import HomePage from '.';

const mockStore = configureMockStore();
const store = mockStore({});
const history = createMemoryHistory({
    initialEntries: [''],
});

test('O Componente <HomePage /> deve ser renderizado na tela corretamente', () => {
    const Element = renderer.create(
        <Router history={history}>
            <Provider store={store}>
                <HomePage />
            </Provider>,
        </Router>,

    ).toJSON();
    expect(Element).toMatchSnapshot();
});
