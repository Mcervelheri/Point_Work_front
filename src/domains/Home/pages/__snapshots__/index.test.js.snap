// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`O Componente <HomePage /> deve ser renderizado na tela corretamente 1`] = `
Array [
  <div
    className="mainContainer"
  >
    <div
      className="ant-page-header pageHeader"
    >
      <div
        className="ant-page-header-content"
      >
        <div
          className="pageHeaderContent"
        >
          <div
            className="ant-page-header dividerNoColor ant-page-header-ghost"
          />
          <div
            className="container"
          >
            <img
              alt="Logotipo AL5 Bank"
              src="logo-al5-dark.svg"
            />
            <p
              className="textWelcome"
            >
              Bem-vindos ao Painel Administrativo do AL5 Bank!
            </p>
            <p
              className="textHelp"
            >
              O que vocÃª gostaria de fazer?
            </p>
            <div
              className="shortcutsContainer"
            >
              <div>
                <button
                  className="button"
                  onClick={[Function]}
                  type="button"
                >
                  <span
                    className="icon"
                    style={
                      Object {
                        "height": 20,
                        "width": 20,
                      }
                    }
                  >
                     
                  </span>
                </button>
                <p
                  className="title"
                >
                  Contas
                </p>
              </div>
              <div>
                <button
                  className="button"
                  onClick={[Function]}
                  type="button"
                >
                  <span
                    className="icon"
                    style={
                      Object {
                        "height": 20,
                        "width": 20,
                      }
                    }
                  >
                     
                  </span>
                </button>
                <p
                  className="title"
                >
                  Recargas
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  ",",
]
`;
